src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyleDiagnosticAnalyzerBase.cs

```diff
@@ -13,8 +13,9 @@
 
 namespace Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
 {
-    internal abstract class NamingStyleDiagnosticAnalyzerBase<TLanguageKindEnum> :
-        AbstractBuiltInCodeStyleDiagnosticAnalyzer where TLanguageKindEnum : struct
+    internal abstract class NamingStyleDiagnosticAnalyzerBase<TLanguageKindEnum>
+        : AbstractBuiltInCodeStyleDiagnosticAnalyzer, IInProcessAnalyzer
+        where TLanguageKindEnum : struct
```

Comment:
why is naming style inproc only?  Seems lke this would be an ideal one to run oop...

Reply:
I'm not sure. We have a hypothesis that .editorconfig didn't work out of proc when it was implemented.